appkit/src/main/java/org/ergoplatform/appkit/RestApiErgoClient.java

```diff
@@ -33,14 +34,23 @@
      *                    form `https://host[:port]` where port is optional.
      *                    If `null` or empty string passed then the Explorer client is not
      *                    initialized and the client works in the `node only` mode.
+     * @param proxy       Requests are passed through this proxy (null it's ok).
```

Comment:
"if non-null" instead of "null it's ok"?

Reply:
Fixed!